File: javascript/alturas.js
const n = parseInt(prompt("Quantas pessoas serao digitadas?"));

let nomes = [], idades = [], alturas = [];

let = alturaMedia = 0, menor16 = 0, porcentagemMenor16 = 0;

for(i = 0; i < n; i++) {
    console.log("Dados da %da pessoa:", i);
    nomes[i] = prompt("Nome:");
    idades[i] = parseInt(prompt("Idade;"));
    alturas[i] = parseFloat(prompt("Altura:"));
}

for(i = 0; i < n; i++) {
    alturaMedia += alturas[i];

    if (idades[i] < 16) {
        menor16++;
    }
}

alturaMedia /= n;
porcentagemMenor16 = (menor16 / n) * 100;

console.log("Altura média: %.2f", alturaMedia);
console.log("Pessoas com menos de 16 anos: %.1f%%", porcentagemMenor16);

for (i = 0; i < n; i++) {
    if (idades[i] < 16) {
        console.log(nomes[i]);
    }
}

File: python/alturas.py
n = int(input("Quantas pessoas serao digitadas? "))

nomes = []
idades = []
alturas = []

soma_alturas = 0
menor_16 = 0

for i in range(0, n):
    print(f"Dados da {i+1}a pessoa:")
    nomes.append(input("Nome: "))
    idades.append(int(input("Idade: ")))
    alturas.append(float(input("Altura: ")))

    soma_alturas += alturas[i]

    if idades[i] < 16:
        menor_16 += 1

altura_media = soma_alturas / n
porcentagem_menor_16 = menor_16 / n * 100

print(f"Altura média: {altura_media}")
print(f"Pessoas com menos de 16 anos: {porcentagem_menor_16}%")

for i in range(0, n):
    if idades[i] < 16:
        print(nomes[i])
